<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestShellClass</class>
 <widget class="QMainWindow" name="TestShellClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test Shell</string>
  </property>
  <widget class="QSplitter" name="globalLayout">
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QSplitter" name="mainLayout">
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Percent position: ______%     Frame ______ of ______</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="FrameView" name="mainFrame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="seekSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QListWidget" name="logText">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="testInfoLayout">
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="FrameView" name="testFrames"/>
    <widget class="QListWidget" name="testText">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="buttonsBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameView</class>
   <extends>QGraphicsView</extends>
   <header>../../Common/Frame/FrameView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
